utils/protocol-types-generator/index.js

```diff
@@ -72,12 +73,47 @@ declare global {
 
 export default Protocol;
 `;
-  const outputPath = path.join(__dirname, '..', '..', 'src', 'protocol.d.ts');
-  require('fs').writeFileSync(outputPath, output);
-  console.log(`Wrote protocol.d.ts for ${version} to ${path.relative(process.cwd(), outputPath)}`);
-  console.log(`You should commit the changes.`);
+
+  return {output, version};
 });
 
+const protocolOutputPath = path.join(__dirname, '..', '..', 'src', 'protocol.d.ts');
```

Comment:
Readability:

```suggestion
const protocolOutputPath = path.join(__dirname, '../../src/protocol.d.ts');
```

Reply:
I'd rather leave this as is so it works should anyone run this on Windows.

Reply:
[`path.join` normalizes](https://nodejs.org/api/path.html#path_path_join_paths), so it would still work on Windows. (I would not have suggested this change otherwise.)